lib/author.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/book'
```

Comment:
Not needed ;)

Reply:
Thanks for this, Kat! I've been doing it wrong the whole time...